updated the requirements.txt

deepthi1107 opened the pull request:
Fixes: issue #213 [updated requirements.txt]

Changes: I have changed the requirements.txt file according to #213 

Screenshots for the change:
![image](https://user-images.githubusercontent.com/79050917/222885690-fb2f28b5-b2f2-4b54-8841-b2219dfbb0ed.png)

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run `./gradlew spotlessCheck` to ensure the code formatting is maintained.

- [x+] Run the unit tests with `./gradlew check` to make sure you didn't break anything

deepthi1107 committed:
updated the requirements.txt

deepthi1107 committed:
added rotation of images as data augmentation process

deepthi1107 committed:
Update README.md

deepthi1107 committed:
Train of yolov5 model

Param-29 commented:
Great work!, could you merge your PR’s I see to many PRs and it misses the point of one PR for one task 
I think you can merge PRs into the following 3 

1. Data Augmentation
    1. Add in the description of all the changes you are making Eg: Rotation, blur, etc etc
2. Image classification  
3. Object detection
    1. Using YOLOv5s & CNN

This would help in getting the bigger picture of what changes you are making 

The above is applicable for following PRs

#253-#301 (~50PRs in total)

deepthi1107 commented:
Hello Param,

I would do the changes and let you know.

Best regards,
Deepthi M

On Sun, 9 Apr 2023 at 11:49 AM, Param Mirani ***@***.***>
wrote:

> Great work!, could you merge your PR’s I see to many PRs and it misses the
> point of one PR for one task
> I think you can merge PRs into the following 3
>
>    1. Data Augmentation
>       1. Add in the description of all the changes you are making Eg:
>       Rotation, blur, etc etc
>    2. Image classification
>    3. Object detection
>       1. Using YOLOv5s & CNN
>
> This would help in getting the bigger picture of what changes you are
> making
>
> The above is applicable for following PRs
>
> #253 <https://github.com/openMF/ppi-vision/pull/253>-#301
> <https://github.com/openMF/ppi-vision/pull/301> (~50PRs in total)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/openMF/ppi-vision/pull/253#issuecomment-1501052104>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AS3DRJOEMQXKU523PE77UBTXAJIINANCNFSM6AAAAAAVPNNVNU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

deepthi1107 commented:
Following are the changes done as suggested by @Param Mirani
<https://mifos.slack.com/team/U04UG3A359N>. Kindly check and do let me know
for further changes and give feedback please.
https://github.com/openMF/ppi-vision/pull/308
https://github.com/openMF/ppi-vision/pull/306
https://github.com/openMF/ppi-vision/pull/304

On Sun, 9 Apr 2023 at 16:02, Deepthi ***@***.***>
wrote:

> Hello Param,
>
> I would do the changes and let you know.
>
> Best regards,
> Deepthi M
>
> On Sun, 9 Apr 2023 at 11:49 AM, Param Mirani ***@***.***>
> wrote:
>
>> Great work!, could you merge your PR’s I see to many PRs and it misses
>> the point of one PR for one task
>> I think you can merge PRs into the following 3
>>
>>    1. Data Augmentation
>>       1. Add in the description of all the changes you are making Eg:
>>       Rotation, blur, etc etc
>>    2. Image classification
>>    3. Object detection
>>       1. Using YOLOv5s & CNN
>>
>> This would help in getting the bigger picture of what changes you are
>> making
>>
>> The above is applicable for following PRs
>>
>> #253 <https://github.com/openMF/ppi-vision/pull/253>-#301
>> <https://github.com/openMF/ppi-vision/pull/301> (~50PRs in total)
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/openMF/ppi-vision/pull/253#issuecomment-1501052104>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AS3DRJOEMQXKU523PE77UBTXAJIINANCNFSM6AAAAAAVPNNVNU>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

deepthi1107 commented:
commit name                         :  PR link
added object detection files : https://github.com/openMF/ppi-vision/pull/308
image classification using yolov5 and cnn layers:
https://github.com/openMF/ppi-vision/pull/306
added data augmentation techniques :
https://github.com/openMF/ppi-vision/pull/304 (edited)

On Mon, 10 Apr 2023 at 18:18, Deepthi ***@***.***>
wrote:

> Following are the changes done as suggested by @Param Mirani
> <https://mifos.slack.com/team/U04UG3A359N>. Kindly check and do let me
> know for further changes and give feedback please.
> https://github.com/openMF/ppi-vision/pull/308
> https://github.com/openMF/ppi-vision/pull/306
> https://github.com/openMF/ppi-vision/pull/304
>
> On Sun, 9 Apr 2023 at 16:02, Deepthi ***@***.***>
> wrote:
>
>> Hello Param,
>>
>> I would do the changes and let you know.
>>
>> Best regards,
>> Deepthi M
>>
>> On Sun, 9 Apr 2023 at 11:49 AM, Param Mirani ***@***.***>
>> wrote:
>>
>>> Great work!, could you merge your PR’s I see to many PRs and it misses
>>> the point of one PR for one task
>>> I think you can merge PRs into the following 3
>>>
>>>    1. Data Augmentation
>>>       1. Add in the description of all the changes you are making Eg:
>>>       Rotation, blur, etc etc
>>>    2. Image classification
>>>    3. Object detection
>>>       1. Using YOLOv5s & CNN
>>>
>>> This would help in getting the bigger picture of what changes you are
>>> making
>>>
>>> The above is applicable for following PRs
>>>
>>> #253 <https://github.com/openMF/ppi-vision/pull/253>-#301
>>> <https://github.com/openMF/ppi-vision/pull/301> (~50PRs in total)
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/openMF/ppi-vision/pull/253#issuecomment-1501052104>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AS3DRJOEMQXKU523PE77UBTXAJIINANCNFSM6AAAAAAVPNNVNU>
>>> .
>>> You are receiving this because you authored the thread.Message ID:
>>> ***@***.***>
>>>
>>
